templates/default/delos.css

```diff
@@ -5910,7 +5910,7 @@ button.close {
   height: 10px;
   margin: 1px;
   text-indent: -999px;
-  border: 1px solid #ffffff;
+  border: 1px solid #fff;
```

Comment:
This stuff is really bad since it is a no change. We should either standardize on the less-compiler or be more picky about changes we include in commits by using `git add -p`.